tensorflow/core/api_def/base_api/api_def_ParallelInterleaveDataset.pbtxt

```diff
@@ -18,5 +84,11 @@ when loading data from a variable-latency datastores (e.g. HDFS, GCS), as it
 allows the training step to proceed so long as some data is available.
 
 !! WARNING !! This dataset is not deterministic!
```

Comment:
I would update this bit to say the it is not deterministic when sloppy is set to `True`.